index.rst

```diff
@@ -80,6 +80,7 @@ for discussion around a future HTTP client PSR.
    :hidden:
    :caption: Components
 
+   components/dependency-graph
```

Comment:
the dependency graph is not a component itself :-)

i would prefer to embed this into some architecture background document that also has some text and not only the image.